docs/guide/pt-BR/usage-with-symfony.md

```diff
@@ -0,0 +1,63 @@
+# Uso com Symfony
+
+<<<<<<< Updated upstream
```

Comment:
![image](https://github.com/yiisoft/db-migration/assets/525501/d56dc0cf-1ae7-4960-b5a7-c78599df6e00)

It looks strange.